### DISCUSSION TITLE
Dark mode in form/table builder

### DISCUSSION THREAD

**Discussion by XarchencoMO** [filamentphp/filament#12536] [Ideas]
in form/table builders v2 was possible to adjust the dark mode, Can you return the ability to adjust the dark mode in v3 ?

**Comment by dissto** [comment]
[dark mode](https://filamentphp.com/docs/3.x/panels/themes#changing-the-default-theme-mode)? 🙃

**Reply by nikspyratos** [reply]
That works for panels, what about forms etc not used in panels?

**Reply by XarchencoMO** [reply]
> [dark mode](https://filamentphp.com/docs/3.x/panels/themes#changing-the-default-theme-mode)? 🙃

I agree with @nikspyratos, this should work in all forms, not just panels

**Reply by leandrocfe** [reply]
https://github.com/filamentphp/filament/blob/3.x/packages/support/tailwind.config.preset.js

Based on this preset, Filament utilizes darkMode: 'class'. Therefore, if you have a parent div with the class="dark", the form will be displayed in dark mode.

**Reply by XarchencoMO** [reply]
@leandrocfe sounds good, but what about reverse case ? if i want to disable dark mode for some form or table ?

**Reply by leandrocfe** [reply]
Provide an example what you are trying to achieve..

**Reply by nikspyratos** [reply]
I'm not too well versed in TW's dark mode handling, but _experientially_ from using a form & wizard outside of panels, it was going with my system setting regardless of if I had forced the rest of the app in light mode or not.

My only solution was to remove dark mode entirely in my app, which TW3 doesn't really support either - I had to set my dark mode selector to something that would never actually be used so that dark mode is never triggered.

**Reply by danharrin** [reply]
If you use our preset, then it won't do that, because dark mode is set to `class` not `system`
